sdk/anomalydetector/Azure.AI.AnomalyDetector/CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Release History
 
-## 3.0.0-preview.4 (Unreleased)
-
+## 3.0.0-preview.4 (2021-12-20)
```

Comment:
We should update this; the library wasn't released on the date listed.   This should reflect the date the package is published.